ui/tests/acceptance/config-ui/messages/messages-test.js

```diff
@@ -126,15 +127,6 @@ module('Acceptance | Enterprise | config-ui/message', function (hooks) {
     await visit('/vault/logout');
   });
 
-  test('it should show an empty state when no messages are created', async function (assert) {
-    await click(CUSTOM_MESSAGES.navLink);
-    assert.dom(GENERAL.emptyStateTitle).exists();
-    assert.dom(GENERAL.emptyStateTitle).hasText('No messages yet');
-    await click(CUSTOM_MESSAGES.tab('On login page'));
-    assert.dom(GENERAL.emptyStateTitle).exists();
-    assert.dom(GENERAL.emptyStateTitle).hasText('No messages yet');
-  });
-
```

Comment:
I'm guessing there just wasn't reliable teardown within the tests to delete custom messages that were created (afterEach)?

Reply:
correct, we tried all the things, but tests continued to remain.